Signal/src/call/PeerConnectionClient.swift

```diff
@@ -139,21 +144,25 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
                                                 delegate: self)
         createAudioSender()
         createVideoSender()
+
+        if callType == .Outgoing {
+            // When placing an outgoing call, it's our responsibility to create the DataChannel. 
+            // Recipient will not have to do this explicitly.
+            createSignalingDataChannel()
+        }
     }
 
     // MARK: - Media Streams
 
-    public func createSignalingDataChannel() {
+    fileprivate func createSignalingDataChannel() {
```

Comment:
I think this should be `private` as we're no longer using it outside of this class.